docs/src/main/sphinx/security/jwt-auth.rst

```diff
@@ -0,0 +1,145 @@
+==================
+JWT authentication
+==================
+
+Trino can be configured to authenticate client access using `JSON web tokens
+<https://en.wikipedia.org/wiki/JSON_Web_Token>`_. A JWT is a small, web-safe
+JSON representation that contains the same cryptographic information as a
+certificate, including:
+
+-  Subject
+-  Valid time period
+-  Signature
+
+A `JWT <jwt.io>`_ is designed to be passed between servers as proof of prior
+authentication. An end-user logs into a client application and requests access
```

Comment:
End user is not hyphenated as a noun (https://developers.google.com/style/word-list?hl=en#letter-e)

Reply:
k, thanks